client/src/app/site/motions/models/category-numbering.ts

```diff
@@ -0,0 +1,22 @@
+import { Motion } from '../../../shared/models/motions/motion';
+
+/**
+ * wrapper class for the HTTP Call
+ */
+export class CategoryNumbering {
+    private motions: number[];
+
+    public constructor() {}
+
+    public setMotions(motionList: Motion[]): void {
+        const motion_id_list: number[] = [];
+        motionList.forEach(motion => {
+            motion_id_list.push(motion.id);
+        });
+        this.motions = motion_id_list;
+    }
+
+    public getMotions(): number[] {
```

Comment:
can just be a getter:
`public get motionIds (...)`

would require to add `_` to the motionIds above